include/stdexec/execution.hpp

```diff
@@ -5736,13 +5736,428 @@ namespace stdexec {
+            requires __callable<_Tag, env_of_t<const _EnvProvider&>, _As...>
+          constexpr _Ret (*operator()(_Tag(*)(_Ret(*)(_As...) noexcept)) const noexcept)(void*, _As...) noexcept {
+          return +[](void* __env_provider, _As... __as) noexcept -> _Ret {
+            static_assert(__nothrow_callable<_Tag, const _EnvProvider&, _As...>);
+            return _Tag{}(get_env(*(const _EnvProvider*) __env_provider), (_As&&) __as...);
+          };
+        }
+      };
+
+    
+    template <class _Sig>
+      struct __storage_vfun;
+
+    template <class _Tag, class... _As>
+      struct __storage_vfun<_Tag(void(*)(_As...))> {
+        void (*__fn_)(void*, _As...);
+        void operator()(_Tag, void* __storage, _As&&... __as) const {
+          return __fn_(__storage, (_As&&) __as...);
+        }
+      };
+
+    template <class _Tag, class... _As>
+      struct __storage_vfun<_Tag(void(*)(_As...) noexcept)> {
+        void (*__fn_)(void*, _As...) noexcept;
+        void operator()(_Tag, void* __storage, _As&&... __as) const noexcept {
+          return __fn_(__storage, (_As&&) __as...);
+        }
+      };
+
+    template <class _Storage, class _T>
+      struct __storage_vfun_fn {
+        template <class _Tag, class... _As>
+            requires __callable<_Tag, __type<_T>, _Storage&, _As...>
+          constexpr void (*operator()(_Tag(*)(void(*)(_As...))) const noexcept)(void*, _As...) {
+          return +[](void* __storage, _As... __as) -> void {
+            return _Tag{}(__type<_T>{}, *(_Storage*) __storage, (_As&&) __as...);
+          };
+        }
+        template <class _Tag, class... _As>
+            requires __callable<_Tag, __type<_T>, _Storage&, _As...>
+          constexpr void (*operator()(_Tag(*)(void(*)(_As...) noexcept)) const noexcept)(void*, _As...) noexcept {
+          return +[](void* __storage, _As... __as) noexcept -> void {
+            static_assert(__nothrow_callable<_Tag, __type<_T>, _Storage&, _As...>);
+            return _Tag{}(__type<_T>{}, *(_Storage*) __storage, (_As&&) __as...);
+          };
+        }
+      };
+
+    struct __get_vtable_t {
+      template <class Storage>
+          requires tag_invocable<__get_vtable_t, const Storage&>
+        const auto operator()(const Storage& __storage) const noexcept {
+          return tag_invoke(__get_vtable_t{}, __storage);
+        }
+    };
+    inline constexpr __get_vtable_t __get_vtable{};
+
+    struct __get_object_pointer_t {
+      template <class Storage>
+          requires __tag_invocable_r<void*, __get_object_pointer_t, const Storage&>
+        void* operator()(const Storage& __storage) const noexcept {
+          return tag_invoke(__get_object_pointer_t{}, __storage);
+        }
+    };
+    inline constexpr __get_object_pointer_t __get_object_pointer{};
+
+    struct __delete_t {
+      template <class _Storage, class _T>
+          // requires tag_invocable<__delete_t, __type<_T>, _Storage&>
```

Comment:
I'm curious why this constraint is commented out.

Reply:
My bad. Forgot to redo that after debugging an issue.